### PR TITLE
Updating README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by pavshr**
Click on a branch to preview the files in GitHub

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+### Submitting an issue
+- Give the issue a short, clear title that describes the bug or feature request
+- Include what version of Smittestopp you are using. If you are reporting a bug, include details on the device you get it:
+  - _**OS Version**_, e.g., iOS 14.4 or Android 9
+  - _**Device**_, e.g., iPhone SE, iPhone 12, Samsung Galaxy S20
+  - **_Language_**, e.g., Norwegian/English
+  - _**Google Play Services version**_, if you the issue affects Android
+- Include steps to reproduce the issue
+- Include screenshots, if possible
+- Mark issue with an appropriate label, e.g., `is:bug`, `affects:ios`
```
Comment (OsirisTerje) [inline CONTRIBUTING.md:19]: Include "Try to", or "if possible" so as not to raise the bar too high

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Use markdown formatting as appropriate to make the issue and code more readable.
+- Do not include any personal information in the issue description and in the screenshots.
+
+### Confirming issues
+If you have resources and possess required devices, you can contribute by confirming that the issue exists. We will try to confirm the issue ourself since we have a good set of hardware to test Smittestopp on. However, due to variety of devices, especially on Android, we are not always able to do it. Feel free to contribute by confirming other people's issue by yourself.
+
+### Working on an issue
+If you want to fix an existing issue, please claim it first by commenting on the GitHub issue that you will work on it. This prevents duplicated efforts from other contributors on the same issue.
+
+You can start working on the Pull Request after the team assigns the issue to you.
```
Comment (OsirisTerje) [inline CONTRIBUTING.md:29]: I don't think we should require assignments before PRs.  Some people just go straight to the PR, and then do the issue afterward.  The sentence before is good enough, asking them to "claim it by commenting" -. that's a good one!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+If you have questions about one of the issues, please comment on them.
+
+## Creating pull requests
+When you get assigned an issue, you can start working on a pull request to address it. Please ensure to do the following steps:
```
Comment (OsirisTerje) [inline CONTRIBUTING.md:34]: Same, drop the assignment thing :-)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+1. Fetch the latest code. 
+2. Create a branch from latest `dev` and give it a proper name, e.g., bugfix/issue-1-crash-on-main-page
+3. Create a draft pull request from `your branch` to `dev` and push your changes regularly. You can also indicate your work in progress using labels and WIP in the pull request title.
+4. Connect issue and pull request on GitHub so the development process is transparent.
+5. Push your code regularly to remote.
```
Comment (OsirisTerje) [inline CONTRIBUTING.md:39]: Under 3 you already have stated they should push regularly.  Perhaps have it only one place?

**Review by jonasofrich — APPROVED** [review]

**Review by nc-kano — APPROVED** [review]

**Pull request merged**